Doc/library/itertools.rst

```diff
@@ -398,7 +398,7 @@ loops that truncate the stream.
 .. function:: filterfalse(predicate, iterable)
 
    Make an iterator that filters elements from iterable returning only those for
-   which the predicate is ``False``. If *predicate* is ``None``, return the items
+   which the predicate is false. If *predicate* is ``None``, return the items
```

Comment:
`False` is also correct here; this change seems to be pedantic churn only, and has no real value, IMO. I'd keep the current text. The following sentence uses ``None``; I don't see any reason not to use ``False`` here. I'd consider leaving this as it is.

Reply:
No, `False` explicitly means the Python object `False`, no other false objects. So that's incorrect here. The `None` case is different.